src/checkout/components/BillingDetails/BillingDetails.tsx

```diff
@@ -0,0 +1,24 @@
+import { Grid, Input, Typography, useTranslate } from "../../../_shared";
+
+export const BillingDetails = () => {
+  const translate = useTranslate();
+
+  return (
+    <Grid container columnSpacing="18px" rowSpacing="24px">
+      <Grid item xs={12}>
+        <Typography variant="subtitle1">
+          {translate("checkout.billing.title")}
+        </Typography>
+      </Grid>
+      <Grid item xs={6}>
+        <Input name={translate("checkout.billing.name")} />
+      </Grid>
+      <Grid item xs={6}>
+        <Input name={translate("checkout.billing.email")} />
+      </Grid>
+      <Grid item xs={6}>
+        <Input name={translate("checkout.billing.phone")} />
```

Comment:
the name of the input can not language-dependent, it is the name of the prop that will be added to the form values object, we neet to split the name from the translated label